src/content/developers/docs/consensus-mechanisms/pos/index.md

```diff
@@ -29,6 +29,18 @@ To participate as a validator, a user must deposit 32 ETH into the deposit contr
 
 Whereas under proof-of-work, the timing of blocks is determined by the mining difficulty, in proof-of-stake, the tempo is fixed. Time in proof-of-stake Ethereum is divided into slots (12 seconds) and epochs (32 slots). One validator is randomly selected to be a block proposer in every slot. This validator is responsible for creating a new block and sending it out to other nodes on the network. Also in every slot, a committee of validators is randomly chosen, whose votes are used to determine the validity of the block being proposed.
 
+## How a Transaction Gets Executed in Ethereum PoS {#transaction-execution-ethereum-pos}
+
+The following provides an overview of how an account's transaction gets executed in Ethereum Proof of Stake, in contrast to [how transactions were mined](/developers/docs/consensus-mechanisms/pow/mining/) in Ethereum PoW.
+
+1. A user creates and signs a [transaction](/developers/docs/transactions/) with an [account's](/developers/docs/accounts/) private key. The user, usually assisted by their wallet, configures the transaction fees that they are prepared to pay in order to include the transaction in a block. These fees go partially to the validator ([tips](/developers/docs/gas/#priority-fee)) or [get burned](/developers/docs/intro-to-ether/#burning-ether) ([base fee](/developers/docs/gas/#base-fee)).
+2. The wallet sends the transaction to an Ethereum [execution client](/developers/docs/nodes-and-clients/) which verifies its validity.
```

Comment:
It doesn't have to originate from a wallet - it could be a user interacting directly with a node, coding in a dev framework, using a web app etc

```suggestion
2. The transaction is submitted to an Ethereum [execution client](/developers/docs/nodes-and-clients/) which verifies its validity. This means ensuring that the sender has enough ETH to fulfil the transaction and they have signed it with the correct key.
```